go.mod

```diff
@@ -99,6 +99,12 @@ require (
 	github.com/go-jose/go-jose/v3 v3.0.3
 )
 
+require (
+	github.com/asaskevich/EventBus v0.0.0-20200907212545-49d423059eef // indirect
+	github.com/emersion/go-sasl v0.0.0-20200509203442-7bfe0ed36a21 // indirect
+	github.com/emersion/go-smtp v0.21.3 // indirect
+)
+
```

Comment:
These dependencies are used in e2e only. I think we should make `./e2e` a separate go module, like `./custombuild`.

Then this PR should make no changes to `<project_root>/go.mod`